todo-list/src/components/TodoList.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+
+const TodoList = (props) => {
+	return (
+		<>
+			{props.tasks.map((task, index) => {
+				return (
+					<div key={index}>
+						<h2> {task.title}</h2>
+						<p> {task.description}</p>
+						<h4> {task.priority}</h4>
+						<button
+							className="button"
```

Comment:
could have made a consistent button to input rather than typing it all out again